layouts/partials/feeback.html

```diff
@@ -0,0 +1,59 @@
+<style>
```

Comment:
Please rename this file to `feedback.html` . 😊

Reply:
😅